app/assets/javascripts/components/ResponseView.coffee

```diff
@@ -36,10 +36,10 @@
       i
         className: 'fa fa-clock-o fa-lg'
       span className: 'date-requested',
-        "Aangevraagd op: #{moment(requestedAt).format('LLL')}",
+        "Aangevraagd op: #{moment(requestedAt).format('LLLL')}",
         br {}
       span className: 'date-created',
-        "Ingevuld op: #{moment(createdAt).format('LLL')}"
+        "Ingevuld op: #{moment(createdAt).format('LLLL')}"
```

Comment:
Prima om maandnaam te gebruiken ipv getal. Lijkt me wel nuttig om de dagnaam te blijven tonen. Het formaat `LLL` staat gedefinieerd in app/assets/javascripts/config/moment.coffee